service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersSubmittedControllerTest.java

```diff
@@ -241,19 +224,9 @@ void shouldSendOrdersByPostWhenOrderIssued() {
     @Test
     void shouldNotifyRepresentativesServedDigitallyWhenOrderIssued() throws NotificationClientException {
         CaseData caseData = caseData();
-        CallbackRequest request = CallbackRequest.builder()
-            .caseDetails(asCaseDetails(caseData))
-            .caseDetailsBefore(asCaseDetails(caseData))
-            .build();
-
-        ManageOrdersEvent generatedOrderEvent = new GeneratedOrderEvent(caseData, ORDER);
-
-        when(manageOrdersEventBuilder.build(extractCaseData(request.getCaseDetails()),
-            extractCaseData(request.getCaseDetailsBefore()))).thenReturn(Optional.of(generatedOrderEvent));
+        postSubmittedEvent(caseData);
 
-        postSubmittedEvent(request);
-
-        verify(notificationClient).sendEmail(
+        verify(notificationClient, timeout(ASYNC_METHOD_CALL_TIMEOUT)).sendEmail(
```

Comment:
I think you can use the checkUntil method here, it is found in `AssertionHelper`. It also means that you don't need to add the throws to the method signature